app/question-component/question.html

```diff
@@ -0,0 +1,7 @@
+<div class="main-question-container">
+  <div class="question-text-container">{{$ctrl.question.text}}</div>
+  <div class="answers-container">
+    <picture-answer class="answer" mg-if="answer.type === 'image'" ng-repeat="answer in $ctrl.question.answers"></picture-answer>
```

Comment:
Both `picture-answer` and `text-answer` take 3 attributes in as bindings; the answer, the question nubmer, and the answer number:

```
<picture-answer answer="answer" question-number="$ctrl.questionNumber" answer-number="$index" ng-if=...>
```

**ALSO**: you probably meant `ng-if`, not `mg-if`